bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableParser.scala

```diff
@@ -49,14 +49,24 @@ object VoTableParser extends VoTableParser {
   /**
    * parse takes an input stream and attempts to read the xml content and convert it to a VoTable resource
    */
-  def parse(catalog: CatalogName, is: InputStream, checkValidity: Boolean = true): CatalogResult = {
+  def parse(catalog: CatalogName, is: InputStream): CatalogResult = {
     // Load in memory (Could be a problem for large responses)
     val xmlText = Source.fromInputStream(is, "UTF-8").getLines().mkString
 
-    (checkValidity, validate(xmlText)) match {
-      case (true, -\/(e))  => \/.left(ValidationError(catalog))
-      case (false, -\/(e)) => \/.right(parse(XML.loadString(xmlText)))
-      case (_, \/-(r))     => \/.right(parse(XML.loadString(r)))
+    validate(xmlText) match {
+      case -\/(e) if catalog.checkValidity => \/.left(ValidationError(catalog))
+      case -\/(e) if catalog == SIMBAD     =>
```

Comment:
Maybe it has to be this way, but there's something about this that isn't totally satisfying.  There's a `checkValidity` flag on the catalog as if we can handle this behavior generically and yet we need the special case `catalog == SIMBAD` anyway.  We could remove the `checkValidity` flag altogether and just check whether the catalog is SIMBAD directly and not lose anything as far as I can see.